src/test/java/teammates/test/cases/webapi/CreateInstructorInCourseActionTest.java

```diff
@@ -23,12 +23,12 @@
 
     @Override
     protected String getActionUri() {
-        return Const.ResourceURIs.COURSE_ADD_INSTRUCTOR;
```

Comment:
Just a note. Don't forget to change the class name `CreateInstructorInCourseActionTest.java`.

Reply:
Done